web/src/components/base/Checkbox.tsx

```diff
@@ -1,39 +1,19 @@
 import styled from '@emotion/styled'
+import { Checkbox as MUICheckbox, FormControlLabel } from '@mui/material'
```

Comment:
🔧 Same as in the other PR.
```suggestion
import { Checkbox as MuiCheckbox, FormControlLabel } from '@mui/material'
```